src/utils.ts

```diff
@@ -95,6 +95,43 @@ export async function openFile(item: FileTreeItem): Promise<void> {
 
 }
 
+export async function deleteGroup(groupProvider: GroupProvider): Promise<boolean> {
+    let groupNames = [];
+    for (let key in groupProvider.groups) {
+        groupNames.push(key);
+    }
+
+    let groupToDelete = await window.showQuickPick(groupNames, {
+        placeHolder: 'Please select the group you would like to delete'
+    });
+
+    groupProvider.deleteGroup(groupToDelete as string);
+    return true;
+}
+
+export async function deleteTabFromGroup(groupProvider: GroupProvider): Promise<boolean> {
+    let groupNames = [];
+    for (let key in groupProvider.groups) {
+        groupNames.push(key);
+    }
```

Comment:
similar thing here, what happens if there are no groups? could be tackled in a different PR but just curious

Reply:
I think this should be fine since it's a for loop, but good to test it out

Reply:
true, but I mainly meant for the user experience. maybe there should be some handling here? would be confusing to show an empty quick prompting to select a group